services/frontend-v3/package.json

```diff
@@ -3,31 +3,28 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
-    "@ant-design/compatible": "^1.0.2",
-    "@ant-design/icons": "^4.0.5",
-    "@reduxjs/toolkit": "^1.3.6",
-    "@testing-library/jest-dom": "5.8.0",
-    "@testing-library/react": "^10.0.4",
-    "@testing-library/user-event": "10.3.2",
-    "antd": "^4.1.4",
+    "@ant-design/icons": "^4.2.1",
+    "@reduxjs/toolkit": "^1.4.0",
+    "@testing-library/jest-dom": "5.11.1",
+    "antd": "^4.4.3",
     "axios": "^0.19.2",
```

Comment:
does updating axios break the app? I think we are very behind with that version

Reply:
I don't think I've updated it

Reply:
I am not sure if you have seen this before https://www.npmjs.com/package/npm-check-updates but it's helpful to use for finding updates. Simply by running  `npx ncu`

Reply:
Ohh, I was just hovering on the packages in package.json, and it told me the latest version... that sounds like a better way to do it haha